index.html

```diff
@@ -7,5 +7,14 @@
     <h1>Project Night Projects from Code for San Jose</h1>
     <p>Small projects for new members to work on.
     <p><a href="instructions.html">Instructions for new members</a>
+    <br>
```

Comment:
You don't need this <br> tag because p tags have space before and after by default.